src/library/scala/collection/immutable/Queue.scala

```diff
@@ -84,6 +84,11 @@ sealed class Queue[+A] protected(protected val in: List[A], protected val out: L
     else if (in.nonEmpty) new Queue(Nil, in.reverse.tail)
     else throw new NoSuchElementException("tail on empty queue")
 
+  override /*IterableLike*/ def forall(p: A => Boolean): Boolean =
```

Comment:
It might generally be a good idea to add comments like this, but since we did not consistently do this in the past, I suggest to remove it.